UPGRADE.md

```diff
@@ -1,5 +1,14 @@
 # API changes
 
+## Version 4.* to 4.10
+
+### Article CSS classes
+
+The CSS classes "first" and "last" are no longer applied to articles. Use CSS
```

Comment:
But aren’t content elements affected too?

Reply:
You are right, we have also removed the classes from CEs in #2226. Did we want to do that?

Reply:
I think so, yes.

Reply:
Fixed in 52641c33acc0f1389999b46164c2b53a7d360fd6 then.